stdlib/list.mli

```diff
@@ -359,6 +359,44 @@ val filteri : (int -> 'a -> bool) -> 'a list -> 'a list
    @since 4.11
 *)
 
+
+(** {1 List manipulation} *)
+
+
+val take : int -> 'a list -> 'a list
+(** [take n l] returns the prefix of [l] of length [n],
+    or a copy of [l] if [n > length l].
```

Comment:
```suggestion
    or a copy of [l] if [n >= length l].
```

Reply:
The versions with `>` are better.  The special case is when `length (take n l) <> n`, i.e. when `n > length l`.

Reply:
reverted